randomsong.py

```diff
@@ -164,17 +165,22 @@ def getRandomSongFromChars(self, verbose=False):
 				print("There are no valid songs. Trying again.")
 
 if __name__ == '__main__':
-	if '-v' in sys.argv:
-		verbose = True
+	
+	os.system("cls")
```

Comment:
Oh this makes the output so much nicer! Could you make this system independent also? The command for linux and bash is `clear` and you should be able to use `os.name` to determine which command to use.